api/CSPViolationReportBody.json

```diff
@@ -0,0 +1,641 @@
+{
+  "api": {
+    "CSPViolationReportBody": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSPViolationReportBody",
+        "spec_url": "https://www.w3.org/TR/CSP3/#cspviolationreportbody",
+        "support": {
+          "chrome": {
+            "version_added": "69"
```

Comment:
Testing in https://mdn-bcd-collector.appspot.com/tests/api/CSPViolationReportBody suggests this isn't shipped yet.

But that's because none of these interface objects are exposed. Without them, one would have to dig through Chromium source to figure out when each one shipped.

Reply:
It appears that this API had actually landed in Chrome 74 as per https://storage.googleapis.com/chromium-find-releases-static/88d.html#88dcf893609cf4916ee33aef1f7658f18f13f760.  @rachelandrew, do you mind updating this?

Reply:
updated